cmd/snapctl/flags.go

```diff
@@ -90,7 +90,7 @@ var (
 
 	flTaskSchedInterval = cli.StringFlag{
 		Name:  "interval, i",
-		Usage: "Interval for the task schedule [ex: 250ms, 1s, 30m]",
+		Usage: "Interval for the task schedule [ex (simple schedule): 250ms, 1s, 30m (cron schedule): \"0 * * * * *\"]",
```

Comment:
We should not overload the interval flag. A new flag should be created for handling cron pattern strings.